darts/models/forecasting_model.py

```diff
@@ -392,6 +411,7 @@ def gridsearch(model_class,
                    series: TimeSeries,
                    covariates: Optional[TimeSeries] = None,
                    forecast_horizon: Optional[int] = None,
+                   epochs: int = 100,
```

Comment:
@hrzn same as with `historical_forecasts`. This time the name is different - would you say it should be `retrain_epochs` for consistency?